web_pwa_oca/static/description/index.html

```diff
@@ -433,8 +433,10 @@ <h1><a class="toc-backref" href="#id3">Usage</a></h1>
 <p>To use your PWA:</p>
 <ol class="arabic simple">
 <li>Open the Odoo web app using a supported browser (like Chrome/Chromium)</li>
-<li>Install the PWA</li>
+<li>Open the browser options</li>
+<li>Click on ‘Add to Home screen’ (or ‘Install’ in other browsers)</li>
```

Comment:
Also remove [this readme line](https://github.com/OCA/web/blame/12.0/web_pwa_oca/readme/INSTALL.rst#L13)